### ISSUE TITLE
PR #476 introduced faulty awk script

### ISSUE THREAD

**Issue by eandriol** [kerl/kerl#483]
#### The bug

In the file `kerl`, as of this writing at line `747`, there now is this:
`BEGIN { printf "%s (See: %s)\n", section, log file }`

In here, please note that `log file` should be `logfile`, or awk will crash/fail on a syntax error.

#### Software versions

irrelevant

#### How to replicate

irrelevant

#### Expected behavior

awk not to crash

#### Additional context

nope

**Comment by paulo-ferraz-oliveira** [comment]
Yeah, I was just looking at this now and pushing to fix 😄 

```
awk: line 1: syntax error at or near file
awk: line 1: syntax error at or near file
awk: line 1: syntax error at or near file
```

Thanks for the report.

**Comment by paulo-ferraz-oliveira** [comment]
For reference, it was here: https://github.com/kerl/kerl/pull/476/files#diff-2df8b125dc6a121ab2da71f2e0fe968fd2d8d710fbecfcb53e93fc2c6c05850bL597

This was a huge pull request and that slipped through the cracks. 👍

**Comment by paulo-ferraz-oliveira** [comment]
Closed via #484.
